sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/models/TrainingFileFilter.java

```diff
@@ -48,21 +48,21 @@ public TrainingFileFilter setPrefix(String prefix) {
      * Get the flag to indicate if sub folders within the set of prefix folders will also need to be included when
      * searching for content to be preprocessed.
      *
-     * @return the includeSubFolders value.
+     * @return the includeSubfolders value.
      */
-    public Boolean isIncludeSubFolders() {
-        return this.includeSubFolders;
+    public Boolean isSubfoldersIncluded() {
+        return this.includeSubfolders;
     }
 
     /**
-     * Set the includeSubFolders flag to indicate if sub folders are also to be included when
+     * Set the includeSubfolders flag to indicate if sub folders are also to be included when
```

Comment:
Might be worth noting that this flag is not supported for training with labels. The service will throw an exception if it set to True in that case

Reply:
@kinelski do you know if we have this info in .NET?